reputation/reputation.go

```diff
@@ -81,6 +82,9 @@ func (rm *Module) AddSource(id string, maddr ma.Multiaddr) error {
 
 // GetTopMiners gets the top n miners with best score
 func (rm *Module) GetTopMiners(n int) ([]MinerScore, error) {
+	if n < 1 {
+		return nil, fmt.Errorf("the number of miners should be greater than zero")
+	}
 	mr := make([]MinerScore, 0, n)
```

Comment:
More defensive code to check that `make` won't panic if the client sends a zero or negative value.